changelogs/2024-09-12_opt-out-of-fast-branch.mdx

```diff
@@ -0,0 +1,21 @@
+---
+_schema: default
+title: Opt out of Fast Branch
```

Comment:
Not sure how specific you want to be here, but the Fast Branch strategy will still be used for the syncing part of the branch. The new option just avoids the part which would allow you to skip rebuilding (this is important if you've got a site that builds differently on your main branch than your feature/staging branches)